### DISCUSSION TITLE
Info per tende somfy

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2450] [GENERALE - Discussioni generiche]
Ciao a tutti, a breve mi installeranno delle tende da sole con motore somfy.
come mi consigliate che integrarle?
il fornitore mi ha proposto radiocomando o a pulsantiera, nel caso di altre esigenze di fornirgli cosa mi occorre.
NB ho impianto bticino smart quindi nel caso la pulsantiera potrei utilizzare un pulsante tende e sarebbe già integrato.

**Comment by Nik71git** [comment]
io le ho integrate tramite il loro trasmettitore a contatto pulito collegato ad un Sonoff 4ch pro con esphome, esattamente questo: https://shop.somfy.it/trasmettitore-radio-rts-a-contatti-puliti.html ma credo che tu riesca a farlo anche con soluzioni Somfy più economiche, vedi la doc dell'integrazione Somfy in HA
